WordPress/src/main/java/org/wordpress/android/widgets/PostListButton.java

```diff
@@ -81,7 +86,16 @@ public void setButtonType(int buttonType) {
 
         mButtonType = buttonType;
         mTextView.setText(getButtonTextResId(buttonType));
-        mImageView.setImageResource(getButtonIconResId(buttonType));
+        if (buttonType == BUTTON_RETRY) {
```

Comment:
I don't think we should do all this just for the retry button (it also still looks different from the other icons) - why don't we get a correctly sized and tinted gridicon to match the others?